lib/IRGen/GenDecl.cpp

```diff
@@ -1338,8 +1338,8 @@ bool LinkEntity::isFragile(ForDefinition_t isDefinition) const {
     auto conformance = getProtocolConformance();
 
     auto conformanceModule = conformance->getDeclContext()->getParentModule();
-    auto isCompletelySerialized = conformanceModule->getResilienceStrategy() ==
-                                  ResilienceStrategy::Fragile;
+    auto isCompletelySerialized =
+        conformanceModule->getASTContext().LangOpts.SILSerializeWitnessTables;
```

Comment:
This is wrong. LinkEntity::isFragile() should be removed, and callers that checked it should always assume 'false'.

Reply:
@slavapestov Slava, LinkEntity::isFragile is currently returning true for e.g. SILFunctions that are [serialized]. Are you sure it is a good idea to always return 'false' for any inputs?